### ISSUE TITLE
Corepack (packageManager) update that should not show

### ISSUE THREAD

**Issue by dougg0k** [raineorshine/npm-check-updates#1485]
Hi,

I enabled Corepack in a project, and it has added the following line outside dependencies.

```
"dependencies": {...},
"packageManager": "pnpm@9.15.0+sha512.76e2379760a4328ec4415815bcd6628dee727a..."
```

After trying to check for updates with `ncu`, it is there as if it were a package, and it wants to add a proper version to it.

```
❯ ncu
Using pnpm
Checking /app/package.json
[====================] 55/55 100%

 eslint-plugin-react                        7.37.2  →  7.37.3
 pnpm                                       9.15.0+sha512.76e2379760a4328ec4415815bcd6628de.....  →  9.15.2
 react-native-screens                       4.3.0  →   4.4.0

Run ncu -u to upgrade package.json
```

Can it be fixed, possibly ignore the `packageManager` property? Or is this intended?

@raineorshine

**Comment by raineorshine** [comment]
Hi, thanks for reporting.

Support for the `packageManager` field was added in #1252 and published in `v16.6.0`.
It was added to the default dependency checks in #1390 and published in `v17.0.0`.

You can ignore `packageManager` by setting `--dep` in the CLI:

```sh
ncu --dep prod,dev,optional
```

...or setting `dep` in your `.ncurc.json` config:

```
{
  "dep": ["prod", "dev", "optional"]
}
```

The npm-check-updates implementation should probably be updated to add the `sha512` suffix, as that is new.

**Comment by dougg0k** [comment]
For update in the project, this was the "fix". `corepack use pnpm@latest`
